### PR TITLE
First go at grid and axis for charts-react-line

### PR DESCRIPTION
**Pull request by seejamescode**
Related to #37 

While the properties change the background grid and axis, it does not yet change the data. We also need to make the axis auto-fit to the data if no properties are provided.

### CODE REVIEW

File: packages/charts-react-axis/src/LeftAxis.js [hunk 1]
```diff
@@ -1,53 +1,70 @@
...
-      label="Label"
+      label={labels[i].toString()}
       offset={offset}
+      width={width}
     />
   );
 };
 /* eslint-enable react/display-name */
 
 const LeftAxis = (props, context) => {
```
Comment (sabov) [inline packages/charts-react-axis/src/LeftAxis.js]: Looks like `BottomAxis` and `LeftAxis` classes have a lot of common functions and properties. Does it make sense to have one class that accepts Axis type (`left` or `bottom`) or maybe one generic class on top?

**Review by sabov — APPROVED** [review]

File: packages/charts-react-axis/src/BottomAxis.js [hunk 2]
```diff
@@ -1,37 +1,46 @@
...
-  strokeWidth: 1,
-  shapeRendering: 'crispEdges',
+/* eslint-disable react/display-name */
+const calcLabels = (max, min, tickCount) => {
+  let labels = [];
+  for (let i = 0; i < tickCount + 1; i++) {
+    labels.push(i * (max - min) / tickCount + min);
+  }
+  return labels;
 };
```
Comment (sabov) [inline packages/charts-react-axis/src/BottomAxis.js]: You can avoid for loop here:
```js
const calcLabels = (max, min, tickCount) => {
    return Array.from(Array(tickCount)).map((_, i) => {
        return i * (max - min) / tickCount + min);
    });
}
```

**Comment by seejamescode** [comment]
@sabov I took your feedback!

File: packages/charts-react-line/src/LineChart.story.js [hunk 3]
```diff
@@ -32,5 +32,15 @@ storiesOf('LineChart', module)
       domainY={extent(data, (d) => d.close)}
       x={(d) => d.date}
       y={(d) => d.close}
+      grid={[
+        {
+          text: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'June', 'July', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
```
Comment (sabov) [inline packages/charts-react-line/src/LineChart.story.js:45]: 🤔  I don't understand how do you convert those values to Date objects to build correct line from the stock.csv values:
```
date,close
24-Apr-07,93.24
25-Apr-07,95.35
26-Apr-07,98.84
27-Apr-07,99.92
30-Apr-07,99.80
```

Comment (seejamescode) [reply packages/charts-react-line/src/LineChart.story.js:45]: I have not connected the flow from data to the grid prop yet. I was planning for that to be the next step!

**Comment by sabov** [comment]
@seejamescode could you resolve the conflicts?

**Pull request merged**